Tests/scs-compatible.yaml

```diff
@@ -36,3 +36,21 @@ iaas:
         url: https://opendev.org/openinfra/interop/src/branch/master/guidelines/2022.11.json
         condition: mandatory
         # Unfortunately, no wrapper to run refstack yet, needs to be added
+  - version: v3
```

Comment:
I'm still puzzled by there being two yaml files. This PR only changes one of them. Why is that (I just don't know)?

Reply:
Moved into a testing directory with https://github.com/SovereignCloudStack/standards/pull/294/commits/e5c0d811b07d76b2941da650267dc0f4944312d6.